.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,22 +1,43 @@
 ---
 name: Bug report
-about: Report a bug on the latest version of Ray.
```

Comment:
revert

Reply:
I'll hard-reset everything outside of `util/sgd` git seems to have messed a bunch of stuff up